src/pages/PassportRedemption/TrackScreen.tsx

```diff
@@ -80,11 +80,11 @@ const Track = ({ setCurrentScreenView }: Props) => {
     }
   };
 
-  const formatTicketCode = (code) => {
-    if (code.length === 3) {
-      setTicketCode(code + '-');
-    }
-  };
+  // const formatTicketCode = (code) => {
+  //   if (code.length === 3) {
+  //     setTicketCode(code + '-');
+  //   }
+  // };
```

Comment:
I think what you wanna do is:
1. If the length is > 3
2. Take the whole string
3. Trim off any non alphanumerics
4. Uppercase
5. And insert the dash at the third index

Reply:
Then all you need to do is run this at every onKeyUp